xrmocap/data_structure/body_model/smplxd_data.py

```diff
@@ -129,17 +129,21 @@ def set_displacement(
                               f'Type: {type(displacement)}.')
             raise TypeError
         if len(displacement.shape) == 1:
+            self.logger.error('Shape of displacement is not correct.\n' +
+                              f'Shape: {type(displacement.shape)}.')
+            raise ValueError
+        elif len(displacement.shape) == 2:
```

Comment:
We specify the shape of the displacement is [batch_size, NUM_VERTS, 3], shall we judge its shape again?

Reply:
Fixed in [49bbead](https://github.com/openxrlab/xrmocap/pull/112/commits/49bbead84901952a229e47c85b2fdb76529ab254).